### PR TITLE
add blogs functionality in profile

### PR DESCRIPTION
**Pull request by fadime-ozdemir**
Add blog posts
Add edit functionalities ( not fully working)

### CODE REVIEW

**Review by halitbatur — APPROVED** [review]

File: src/components/PersonalBlogPosts/index.jsx [hunk 1]
```diff
@@ -3,15 +3,21 @@ import { Row, Button } from "react-bootstrap";
...
-  ];
+  const [posts, setPosts] = React.useState([]);
+  const _URL =
+    "https://public-api.wordpress.com/wp/v2/sites/nobarriers650349093.wordpress.com/posts";
+
+  React.useEffect(() => {
+    const fetchPosts = async () => {
+      const data = await fetch(_URL);
+      const fetchedPosts = await data.json();
+      setPosts(fetchedPosts.slice(0, 3));
```
Comment (halitbatur) [inline src/components/PersonalBlogPosts/index.jsx]: no magic numbers

**Comment by fadime-ozdemir** [comment]
PTAL

**Review by halitbatur — APPROVED** [review]
lets update it with latest dev branch and then we can merge

**Review by khaldoon9kh — APPROVED** [review]
LGTM

**Comment by fadime-ozdemir** [comment]
We decided to delete the blog Section in profile because the blog posts are made only by one person which is the no barriers admin
PTAL

**Comment by halitbatur** [comment]
@fadime-ozdemir sorry can u update it once more ?

**Comment by fadime-ozdemir** [comment]
PTAL

**Review by khaldoon9kh — APPROVED** [review]
LGTM
Just maybe update the snapshot and it will pass the tests

**Comment by fadime-ozdemir** [comment]
PTAL

**Pull request merged**